batch/test-locally.sh

```diff
@@ -24,6 +27,8 @@ else
     export CLOUD_SQL_CONFIG_PATH=/batch-secrets/batch-test-cloud-sql-config.json
 fi
 
+temp_table=$(python3 -c "from batch.server.database import Database; db = Database.create_synchronous(\"$CLOUD_SQL_CONFIG_PATH\"); print(db.temp_table_name_sync(\"foo\"))")
```

Comment:
This is to put the infrastructure in for creating temporary tables and then cleaning them up. I could just generate a uid with `../generate_uid` and not check it's in the table. Would probably be simpler, but still need the trap command above. 

Open to suggestions on how to better do this.